Triangle/carlsProblem/Triangle.java

```diff
@@ -0,0 +1,53 @@
+package carlsProblem;
+
+import java.util.ArrayList;
+
+public class Triangle extends Polygon {
+
+	public static final int TRIANGLE_SIDES = 3;
+	
+	public Triangle(int numOfSides) {
+		super(numOfSides);
+		// TODO Auto-generated constructor stub
+	}
```

Comment:
we don't need to have numOfSides here since numOfSides has to be 3 in a triangle. 
write a `public Triangle()` that supers the Polygon's constructor.
you can simply delete this constructor since you already have one  `public Triangle()` below.